Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM microsoft/terraform-test:0.11.1
```

Comment:
Dockerfile not needed since it is same as microsoft/terraform-test

Reply:
The docker file is needed as we are supplying for external developers a way to test their changes in a local container, so they will need this file to create a local image for their docker container.